### PR TITLE
Updated detailed description - AWS Guard Duty

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [x] Ready

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CRTX-61818

## Description
- Updated the detailed description
- Added a tooltip for the severity

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Documentation

### CODE REVIEW

File: Packs/AWS-GuardDuty/ReleaseNotes/1_2_9.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### AWS - GuardDuty
+- Documentation and metadata improvements.
```
Comment (ShahafBenYakir) [inline Packs/AWS-GuardDuty/ReleaseNotes/1_2_9.md:4]: In general (and here too) we want to refrain from writing this sentence as it confuses customers. Instead, let's give a very short summary of the stuff we added.

**Review by ShahafBenYakir — APPROVED** [review]

File: Packs/AWS-GuardDuty/ReleaseNotes/1_2_9.md (hunk 1 above)
Comment (Shellyber) [reply Packs/AWS-GuardDuty/ReleaseNotes/1_2_9.md:4]: Btw, this is generated by default in the update-release-notes command in the SDK

Comment (ShahafBenYakir) [reply Packs/AWS-GuardDuty/ReleaseNotes/1_2_9.md:4]: What do you mean by default?

Comment (Shellyber) [reply Packs/AWS-GuardDuty/ReleaseNotes/1_2_9.md:4]: When running `demisto-sdk update-release-notes -i Packs/pack-name -u documentation` this is generated automatically to the RN

**Comment by ShirleyDenkberg** [comment]
@ShahafBenYakir Doc review completed.
@Shellyber What about the integration readme?  The parameters should be defined there also. (The table in Step 3.  A new column should be added for Description.)

**Comment by Shellyber** [comment]
@ShirleyDenkberg Will add it now, thanks :)

**Pull request merged**